examples/stem_histogram.py

```diff
@@ -0,0 +1,85 @@
+import click
```

Comment:
If you aren't using click I don't think you need to import the module.

Reply:
Thanks, the line has been removed.